models/datamessage.go

```diff
@@ -0,0 +1,5 @@
+package models
+
+type DataMessage struct {
+	Message string `json:"message,omitempty"`
+}
```

Comment:
What does this represent?

Reply:
This was just created as a way to check that we were able to read and write from mongoDB, with a simple implementation - it has no meaning.